client/src/pages/LoginPage.js

```diff
@@ -1,30 +1,64 @@
-import React from "react";
+import { React, useState, useRef, useEffect } from "react";
 import NavBar from "../components/Navbar";
+import axios from "../components/api";
+import Cookies from "js-cookie";
+import ErrMsg from "../components/ErrorMessage";
 
+const LOG_URL = "/auth/login";
 const LoginPage = () => {
+  const emailRef = useRef();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errMsg, setErrMsg] = useState('');
+
+  useEffect(() => {
+    emailRef.current.focus();
+  }, [])
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post(LOG_URL, { email, password });
+      Cookies.set('user_id', response.data.user.id);
```

Comment:
- [ ] Set cookies as 'user'. No need to break it down. 
Just use JSON.parse() so that it can be saved.